recipes/sdl2_mixer/2.0.4/conanfile.py

```diff
@@ -0,0 +1,134 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class SDL2MixerConan(ConanFile):
+    name = "sdl2_mixer"
+    description = "SDL_mixer is a sample multi-channel audio mixer library"
+    topics = ("sdl2_mixer", "sdl_mixer", "sdl2", "mixer", "audio", "multimedia", "sound", "music")
+    url = "https://github.com/bincrafters/community"
+    homepage = "https://www.libsdl.org/projects/SDL_mixer/"
+    license = "Zlib"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "cmd": [True, False],
+               "wav": [True, False],
+               "flac": [True, False],
+               "mpg123": [True, False],
+               "mad": [True, False],
+               "ogg": [True, False],
+               "opus": [True, False],
+               "mikmod": [True, False],
+               "modplug": [True, False],
+               "fluidsynth": [True, False],
+               "nativemidi": [True, False],
+               "tinymidi": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": True,
+                       "cmd": False,  # needs sys/wait.h
+                       "wav": True,
+                       "flac": True,
+                       "mpg123": True,
+                       "mad": True,
+                       "ogg": True,
+                       "opus": True,
+                       "mikmod": True,
+                       "modplug": True,
+                       "fluidsynth": True,
+                       "nativemidi": True,
+                       "tinymidi": True}
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+    _cmake = None
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        if self.settings.os != "Linux":
+            del self.options.tinymidi
+        else:
+            del self.options.nativemidi
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def requirements(self):
+        self.requires("sdl2/2.0.14@bincrafters/stable")
+        if self.options.flac:
+            self.requires("flac/1.3.3")
+        if self.options.mpg123:
+            self.requires("mpg123/1.26.4")
+        if self.options.mad:
+            self.requires("libmad/0.15.1b")
+        if self.options.ogg:
+            self.requires("ogg/1.3.4")
+            self.requires("vorbis/1.3.7")
+        if self.options.opus:
+            self.requires("opus/1.3.1")
+            self.requires("opusfile/0.12")
+        if self.options.mikmod:
+            self.requires("libmikmod/3.3.11.1")
+        if self.options.modplug:
+            self.requires("libmodplug/0.8.9.0")
+        if self.options.fluidsynth:
+            self.requires("fluidsynth/2.1.5@bincrafters/stable")
```

Comment:
```suggestion
            self.requires("fluidsynth/2.2.2@bincrafters/stable")
```
I'm not sure if it works, but it's the last version for which we have a recipe